.coderabbit.yaml

```
@@ -85,22 +85,4 @@ tools:
-      - "p/javascript"   # JavaScript language rules
-      - "p/security-audit"  # Security best practices
-      - "p/owasp-top-ten"   # OWASP top 10 vulnerabilities
-  gitleaks:
-    rules:
-      - "default"
-      - "github"
-      - "npm"
-      - "aws"
-      - "google"  
\ No newline at end of file
+  
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)